cmake/std/atdm/cts1/custom_builds.sh

```diff
@@ -0,0 +1,36 @@
+#
+# Custom builds for cee-rhel6 env
+#
+# NOTE: This file gets sourced in atdm/utils/set_build_options.sh before the
+# default grep logic is applied.
+#
+
+# Custom compiler selection logic
+
+# TODO: Check with EMPIRE developers about default compiler/mpi ver
```

Comment:
The defaults are INTEL-16.0_OPENMPI-1.10 but it was decided to use intel-18.0.2-openmpi-2.0.3 as the default.